WordPress/src/main/java/org/wordpress/android/util/ViewUtils.kt

```diff
@@ -35,3 +38,43 @@ fun View.expandTouchTargetArea(@DimenRes dimenRes: Int, heightOnly: Boolean = fa
         parent.touchDelegate = TouchDelegate(touchTargetRect, this)
     }
 }
+
+/**
+ * Ensures that the keyboard is only shown when the view has received focused.
+ *
+ * https://developer.squareup.com/blog/showing-the-android-keyboard-reliably/
+ */
+fun View.focusAndShowKeyboard() {
```

Comment:
👍 already reviewed this class when checking the other PR, agreed this is useful 🙇